File: Com.Fubei.Api.OpenApi/Biz/MerchantApi/IFubeiMerchantApi.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using Com.Fubei.Api.OpenApi.Biz.MerchantApi.Model.Request;
using Com.Fubei.Api.OpenApi.Biz.MerchantApi.Model.Response;
using Com.Illuminati.Galileo.Attr;
using Com.Illuminati.Galileo.Biz.MerchantApi.Model.Request;
using Com.Illuminati.Galileo.Biz.MerchantApi.Model.Response;

namespace Com.Illuminati.Galileo.Biz.MerchantApi
{
    /// <summary>
    /// 商户API接口
    /// </summary>
    public partial interface IFubeiMerchantApi
    {
        [MerchantApi(Method = "openapi.payment.order.swipe")]
        OrderPayResultEntity OrderSwipe([JsonObjectParam]OrderPayParam param);

        [MerchantApi(Method = "openapi.payment.order.scan")]
        OrderScanResultEntity OrderScan([JsonObjectParam] OrderScanParam param);

        [MerchantApi(Method = "openapi.payment.order.query")]
        OrderQueryEntity OrderQuery([JsonObjectParam] OrderQueryParam param);
    }
}

File: Com.Fubei.OpenApi.Vb/Biz/MerchantApi/IFubeiMerchantApi.vb
Imports Com.Fubei.Api.OpenApi.Biz.V1.Model.Request
Imports Com.Fubei.Api.OpenApi.Biz.V1.Model.Response
Imports Com.Illuminati.Galileo.Attr

Namespace Biz.V1
    ' 开放平台1.0接口，对应文档
    ' http://docs.51fubei.com/open-api/
    Public Partial Interface IFubeiMerchantApi

        ' 付款码支付
        <MerchantApi(Method:="openapi.payment.order.swipe")>
        Function OrderSwipe(<JsonObjectParam> ByVal param As OrderPayParam) As OrderPayResultEntity

        ' 扫码支付
        <MerchantApi(Method:="openapi.payment.order.scan")>
        Function OrderScan(<JsonObjectParam> ByVal param As OrderScanParam) As OrderScanResultEntity

        ' 订单查询
        <MerchantApi(Method:="openapi.payment.order.query")>
        Function OrderQuery(<JsonObjectParam> ByVal param As OrderQueryParam) As OrderQueryEntity
    End Interface

End Namespace
